### PR TITLE
Add proc.Comm support

### PR DESCRIPTION
**Pull request by ericchu94**
This will give us a way to get the command name of a process without reading the entire stat file.

### CODE REVIEW

File: proc.go [hunk 1]
```diff
@@ -116,6 +116,22 @@ func (p Proc) CmdLine() ([]string, error) {
 	return strings.Split(string(data[:len(data)-1]), string(byte(0))), nil
 }
 
+// Comm returns the command name of a process.
+func (p Proc) Comm() (string, error) {
+	f, err := os.Open(p.path("comm"))
```
Comment (grobie) [inline proc.go]: The indentation seems to be off by one level. Please run `go fmt` and change this commit.

File: proc.go [hunk 2]
```diff
@@ -116,6 +116,22 @@ func (p Proc) CmdLine() ([]string, error) {
...
+		return "", err
+	}
+	defer f.Close()
+
+	data, err := ioutil.ReadAll(f)
+	if err != nil {
+		return "", err
+	}
+
+    return strings.TrimSpace(string(data)), nil
```
Comment (grobie) [inline proc.go]: Oh, or this one is off by using spaces instead of a tabstop?

**Comment by grobie** [comment]
Thank you! Wonderful commit, almost perfect :) I'll merge once the formatting is fixed (please keep all changes in one commit).

**Comment by ericchu94** [comment]
Oops, my bad. I have run `go fmt`

**Comment by grobie** [comment]
I updated the CI setup, the tests will fail in the future if the code is not formatted correctly: https://github.com/prometheus/procfs/commit/9b72e51f46558a3563d5db3d6af228a7947c33c7.

**Pull request merged**